src/plugins/auto/executable_network.cpp

```diff
@@ -699,18 +699,42 @@ InferenceEngine::Parameter MultiDeviceExecutableNetwork::GetMetric(const std::st
                 lock.unlock();
                 if (deviceInfo.deviceName.find("GPU") != std::string::npos) {
```

Comment:
as we discussed better check if the OPTIMAL_BATCH_SIZE metric is supported... any device that supports that will trigger the autobatching (whne the metric value >1), this is how the a-batching is triggered in the core... we should be aligned

Reply:
other wise the auto-batching wouldn't be triggered for the CUDA plugin

Reply:
yeah...makes sense. updated, sorry for misunderstanding of the usage of this metric in the last discussion.